rdtools/test/irradiance_rescale_test.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+from pandas.testing import assert_series_equal
+from rdtools import irradiance_rescale
+from rdtools.normalization import ConvergenceError
+import pytest
+
+
+@pytest.fixture
+def simple_irradiance():
+    times = pd.date_range('2019-06-01 12:00', freq='15T', periods=5)
+    time_series = pd.Series([1,2,3,4,5], index=times)
+    return time_series
+
+
+@pytest.mark.parametrize("method", ['iterative', 'single_opt'])
+def test_rescale(method, simple_irradiance):
+    # test basic functionality
+    modeled = simple_irradiance
+    measured = 1.05 * simple_irradiance
+    rescaled = irradiance_rescale(measured, modeled, method=method)
+    expected = measured
+    assert_series_equal(rescaled, expected, check_exact=False)
+    
+
+def test_warn(simple_irradiance):
```

Comment:
If we don't change the default behavior, a more descriptive name for this test would be helpful.
e.g. test_method_equals_none_warns
Or include a comment explaining the test

Reply:
I'd say making `method='iterative'` the default with no warning is appropriate in 2.0.  Happy to add that to this PR and remove this test if people agree.